.github/workflows/release.yml

```diff
@@ -90,3 +91,23 @@ jobs:
         env:
           CLOUDSMITH_API_KEY: ${{ secrets.CLOUDSMITH_API_KEY }}
         run: make apt-release
+        
+  upgrade-version-config-json:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: pocket-apps/action-update-version@v1
+        with:
+          files: 'packaging/hassio/config.json'
+          version-regexp: '\d+.\d+.\d+'
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Wie gesagt: ich würde das ungern committen. Brauchts ohne commit wirklich das Token? Ausgecheckt ist die Datei ja schon?

Reply:
https://github.com/pocket-apps/action-update-version
By supplying the repo-token the commit will use the release information: author and branch.
wenn alternativ die der branch-name angegeben wird muss auch der Name und die email angegeben werden damit das funktioniert:

<html><body>
<!--StartFragment-->

branch-name | Default branch name to push changes if not repo-token is provided | Release target branch
-- | -- | --
author-name | Commit author name | Release author name
author-email | Commit author email | Release author email

<!--EndFragment-->
</body>
</html>

Reply:
Und wie verhindern wir dass das committed wird?

Reply:
Was meinst du genau? Was möchtest Du verhindern?

Reply:
entweder:
```
      - uses: pocket-apps/action-update-version@v1
        with:
          files: 'packaging/hassio/config.json'
          version-regexp: '\d+.\d+.\d+'
          repo-token: ${{ secrets.GITHUB_TOKEN }}
```
oder (dieses habe ich nicht zum laufen bekommen!)
```
      - uses: pocket-apps/action-update-version@v1
        with:
          files: 'packaging/hassio/config.json'
          version-regexp: '\d+.\d+.\d+'
          branch-name: master
          author-name: evcc-io
          author-email: xyz@evcc
```

Reply:
> Was möchtest Du verhindern?

Ich möchte, dass das config.json template hier im Repo nicht jedesmal neu eingecheckt wird.

Reply:
Siehe https://github.com/pocket-apps/action-update-version/issues/30

Reply:
Tatsächlich sollte es dafür gar keine action brauchen, sondern ein einfaches `sed` sollte reichen?